notebooks/Tutorial6-Time_Series.py

```diff
@@ -0,0 +1,234 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.14.2
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# # Tutorial 6: Time Series
+
+# In this tutorial, we will perform the avatarization of data that contains time series. The approach presented here can be used to anonymize time series that contain a small number of data points.
+
+# ## Connection
+
+# +
+import os
+
+url=os.environ.get("AVATAR_BASE_URL")
+username=os.environ.get("AVATAR_USERNAME")
+password=os.environ.get("AVATAR_PASSWORD")
+
+# +
+# This is the client that you'll be using for all of your requests
+from avatars.client import ApiClient
+from avatars.models import AvatarizationJobCreate, AvatarizationParameters, ImputationParameters, ImputeMethod, ExcludeCategoricalParameters, ExcludeCategoricalMethod, RareCategoricalMethod
+from avatars.models import ReportCreate
+
+from avatars.api import AvatarizationPipelineCreate
+from avatars.processors.proportions import ProportionProcessor
+from avatars.processors.group_modalities import GroupModalitiesProcessor
+from avatars.processors.relative_difference import RelativeDifferenceProcessor
+from avatars.processors.perturbation import PerturbationProcessor
+from avatars.processors.expected_mean import ExpectedMeanProcessor
```

Comment:
In the future, we shall export all the processors in an `__init__.py` file so that the user only has to do:

```python
from avatars.processors import ExpectedMeanProcessor
```

instead of
```python
expected_mean.ExpectedMeanProcessor
```